src/depends/libDatabase/LevelDB.cpp

```diff
@@ -99,6 +75,14 @@ leveldb::Slice toSlice(boost::multiprecision::uint256_t num)
     return (leveldb::Slice)h.ref();
 }
 
+string LevelDB::GetDBName() 
+{
+        if (LOOKUP_NODE_MODE)
```

Comment:
For consistency with existing style, please enclose branched code in braces, even if it's a single line.
Reference: https://github.com/Zilliqa/Zilliqa/wiki/Coding-Guidelines#formatting-and-coding-style